tests/test_trace.py

```diff
@@ -171,5 +172,40 @@ def test_two_level_flattened_hf_bert():
     assert not find_module_in_graph(sub_sch.mod.graph, "attention.self.query")
 
 
+def test_dynamo():
+    from transformers import AutoConfig, BertLMHeadModel
+
+    config = AutoConfig.from_pretrained("bert-large-uncased")
+    with slapo.init_empty_weights():
+        model = BertLMHeadModel(config)
+    sch = slapo.create_schedule(model)
+    subsch = sch[f"bert.encoder.layer.{0}.attention.self"]
```

Comment:
The same as `subsch = sch["bert.encoder.layer.0.attention.self"]`, why we need the string formatting here?

Reply:
I'll fix this. I just copied from other test cases.